ui/app/components/feedback/HumanFeedbackForm.tsx

```diff
@@ -0,0 +1,179 @@
+import { useConfig } from "~/context/config";
```

Comment:
The only changes I made to this component:

- Removed the `Form` wrapper and `_action` field so we can render it directly in the route
- Added a `formError` prop and rendered it below the form to the left of the submit button